### PR TITLE
ref(onboarding docs): [1/4] convert react.tsx

### PR DESCRIPTION
**Pull request by michellewzhang**
relates to https://github.com/getsentry/sentry/issues/61266

### CODE REVIEW

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 1]
```diff
@@ -1,209 +1,171 @@
...
-  if (otherConfigs.length > 0) {
-    sentryInitContent = sentryInitContent.concat(otherConfigs);
-  }
+const getVerifyReactSnippet = () => `
+return <button onClick={() => methodDoesNotExist()}>Break the world</button>;
+        `;
+
+const onboarding: OnboardingConfig = {
+  introduction: () => [],
+  install: (_params: Params) => [
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/javascript/react.tsx:59]:
```suggestion
  install: () => [
```

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 2]
```diff
@@ -1,209 +1,171 @@
...
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  verify: () => [
+    getUploadSourceMapsStep({
+      guideLink: 'https://docs.sentry.io/platforms/javascript/guides/react/sourcemaps/',
+    }),
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/javascript/react.tsx:113]: nit: would probably put it under `configure` but it does not make a difference for now.

**Review by ArthurKnaus — APPROVED** [review]
Awesome! 👍

File: static/app/gettingStartedDocs/javascript/react.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/gettingStartedDocs/javascript/react.tsx]: updated this for the astro file as well!

**Pull request merged**